### ISSUE TITLE
Export to Markdown

### ISSUE THREAD

**Issue by vrodriguezf** [mo1ein/TelegramArchive#8]
Again, in order to use the exported chats as context for LLMs, sometimes it's handy to have the format in Markdown instead of JSON.

**Comment by vrodriguezf** [comment]
I'll submit a PR later

**Comment by mo1ein** [comment]
> Again, in order to use the exported chats as context for LLMs, sometimes it's handy to have the format in Markdown instead of JSON.

Good idea. Exporting to HTML (similar to the Telegram Desktop export that allows viewing messages in a browser) is currently in the TODO state.

**Comment by vrodriguezf** [comment]
I'll submit a PR later.
